consensus/core/src/commit.rs

```diff
@@ -207,23 +207,38 @@ impl Deref for TrustedCommit {
     }
 }
 
+/// A commit that has been synced and certified by a quorum of authorities. It has to be noted that it is expected
+/// in the majority of the cases the `votes` vector to be empty as during the sync commit process we expect to certify
+/// the tip of the fetched commits from an authority which inherently certifies the previous commits.
 #[derive(Clone, Debug)]
 pub(crate) struct CertifiedCommit {
     commit: Arc<TrustedCommit>,
     blocks: Vec<VerifiedBlock>,
+    #[allow(unused)]
```

Comment:
Is `votes` really unused?
Also, just an idea, what about creating 
```
pub(crate) struct CertifiedCommits {
    commits: Vec<CertifiedCommit>,
    votes: Vec<VerifiedBlock>,
}
```
instead of changing `CertifiedCommit`?

Reply:
Good point - not unused now. I like the idea of the `CertifiedCommits` struct.